docs/standard/serialization/system-text-json/immutability.md

```diff
@@ -47,11 +47,13 @@ For types that are immutable because all their property setters are non-public,
 
 ## Non-public property accessors
 
-`System.Text.Json` doesn't support serialization of non-public properties. However, you can enable use of a non-public property *accessor* by using the [[JsonInclude]](xref:System.Text.Json.Serialization.JsonIncludeAttribute) attribute, as shown in the following example:
+You can enable use of a non-public *accessor* on a property by using the [[JsonInclude]](xref:System.Text.Json.Serialization.JsonIncludeAttribute) attribute, as shown in the following example:
 
 :::code language="csharp" source="snippets/how-to-5-0/csharp/NonPublicAccessors.cs" highlight="10,13":::
 :::code language="vb" source="snippets/how-to-5-0/vb/NonPublicAccessors.vb" :::
 
+In .NET 8 and later versions, you can also user the [[JsonInclude]](xref:System.Text.Json.Serialization.JsonIncludeAttribute) attribute to opt non-public members into the serialization contract for a given type.
+
```

Comment:
I didn't include this in the Preview 7 announcement, but one caveat to the above is in the source generator the feature is limited to non-public members that are visible to the generated `JsonSerializerContext` (i.e. it works with internal members in the same project but not with private members or internal members in other assemblies).

Reply:
@eiriktsarpalis Thanks. So in reflection mode, is it any non-public member?

Reply:
Correct, it will work with anything.